### ISSUE TITLE
Add an editable name for each session

### ISSUE THREAD

**Issue by altingrc** [umami-software/umami#2990]
### Describe the feature or enhancement

We have some prospects who frequently visit our website before or after a deal. Now that we can track their session paths and have nice profile images, we’d like to rename some of them to avoid confusion about who is who. It would be helpful to have an editable property for this purpose.

Maybe like this : 
![Screenshot 2024-10-02 at 19 59 21](https://github.com/user-attachments/assets/26f6637e-8613-4948-82a0-c01291b1a568)

And here : 
![Screenshot 2024-10-02 at 19 55 56](https://github.com/user-attachments/assets/34e29c1e-ad9e-4dd2-b564-1d2f732a7b89)

**Comment by mikecao** [comment]
You can already do this by sending session data. See https://umami.is/docs/tracker-functions#session-data

**Comment by altingrainca** [comment]
> You can already do this by sending session data. See https://umami.is/docs/tracker-functions#session-data

Hmm yeah, that's cool. I was thinking about doing it from the dashboard and being able to edit it if we want.

**Comment by mikecao** [comment]
Yes, we should definitely allow editing of session data from the UI

**Comment by rohitmore07** [comment]
Hey @mikecao,
I am very keen to work on this issue. I would be really happy if you could assign this issue to me.

**Comment by franciscao633** [comment]
Cleaning out old enhancements not currently in the pipeline. Will revisit in the future.
